default_conf.json.in

```diff
@@ -26,6 +26,14 @@
         "service_base_port": 8888,
         "service_provider_id": 88
       }
+    },
+    "Logging": {
+      "log": {
+        "type": "console",
+        "file": "../../logs/chronovisor_logfile.txt",
```

Comment:
too many relative path could be risky. it is very likely that "../../logs" does not exist.

Reply:
i agree with Kun. configfile shoiuld have full file path.  full file path in the config file can be generated by the a job  that starts it if needed . for now just use current directory